backend/src/v5/utils/responseCodes.js

```diff
@@ -22,6 +22,7 @@ const ResponseCodes = {};
 
 ResponseCodes.templates = {
 	ok: { message: 'OK', status: 200 },
+	found: { message: 'Found', status: 302 },
```

Comment:
this isn't an actual respond we send out and only used within tests to get the status

so just use 302 instead of this